### PR TITLE
doc: Add DB schema info.

### PR DESCRIPTION
**Pull request by j0sh**

### CODE REVIEW

File: database.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+id |  INTEGER PRIMARY KEY| The ID of the on-chain job
+recordedAt | STRING DEFAULT CURRENT_TIMESTAMP | Time this row was inserted, *after* the job transaction has been confirmed on chain.
+streamID | STRING | Stream identifier, composed of `<NodeID><VideoID><Rendition>`.
+segmentPrice | INTEGER | The cost of transcoding each segment, in wei.
+transcodeOptions | STRING | [Encoded version of the rendition strings](https://github.com/livepeer/go-livepeer/blob/fac0393950e821f16555a17eb460b8abe74f6ada/common/videoprofile_ids.go#L5-L15).
+broadcaster | STRING | Broadaster's Eth address.
+transcoder | STRING | Transcoder's Eth address. Should be your node's address.
+startBlock | INTEGER | Block height from when the job was submitted on-chain.
+endBlock | INTEGER | Block height at which the job expires.
+stopReason | STRING | Reason for stopping a job on the transcoder. This may be before `endBlock` is reached. **Deprecated**
```
Comment (yondonfu) [inline database.md]: This is listed as deprecated, but don't we still set the stop reason in a few places (i.e. https://github.com/livepeer/go-livepeer/blob/4589a1364fa9d29e9d196d259f1f235116d45953/eth/eventservices/jobservice.go#L133)?

File: database.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Column | Type | Description
+--- | --- | ---
+jobID |  INTEGER NOT NULL | The job this receipt belongs to.
+claimID | INTEGER | The claim in which this receipt was submitted at. May be NULL if a claim was not yet submitted.
+seqNo | INTEGER NOT NULL  | Sequence  number of this receipt.
+bcastFile | STRING | Location on disk of the broadcast file. After verification succeeds, this file may not exist anymore.
+bcastHash | STRING | Hash of the source segment from the broadcaster.
+bcastSig | STRING | Broadcaster's signature verifying this segment, approximately `Sign(broadcasterKey, Hash(streamID \| seqNo \| bcastHash))` 
+transcodedHash | STRING | Hash of the concatenation of all transcoded outputs for this segment,
+ `Hash(Hash(Transcode(Seg, Profile1)) \| ... \| Hash(Transcode(Seg, ProfileN)))`
```
Comment (yondonfu) [inline database.md]: Just a minor note on how this is rendered - looks like this line ended up in its own row

**Review by yondonfu — COMMENTED** [review]
Not as relevant right now, but I wonder if in the future if we add more node operating or developer docs in this repo if this should go into a `docs` folder along with other docs

File: database.md (hunk 1 above)
Comment (j0sh) [reply database.md]: With the current (old) network protocol, the transcoder initiates the connection, so it needs heuristics to know when not to retry a job. With the new network, the transcoder doesn't really have the notion of stopping a job; it just receives segments and checks for the validity of each segment from there. As as result, more types of errors become recoverable, such as 'insufficient deposit'.

Perhaps we could continue using `stopReason` to short-cut certain checks by adding a "hard stop" for the entire job (eg , [inconsistent broadcasters](https://github.com/livepeer/go-livepeer/issues/371)), but I think that most of the existing error cases would go away with the new network.

**Comment by j0sh** [comment]
Good thinking, I do have plans to write a more formal API spec, so there's more documentation coming. Moved this file into `docs/`.

File: database.md (hunk 2 above)
Comment (j0sh) [reply database.md]: Fixed 👍

File: database.md (hunk 1 above)
Comment (j0sh) [reply database.md]: Removed the deprecated note, since it's not actually deprecated yet.

**Review by yondonfu — APPROVED** [review]
LGTM 🚢

**Pull request merged**